packages/conventional-changelog-angular/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "conventional-changelog angular preset",
   "main": "index.js",
   "scripts": {
-    "lint": "jshint *.js --exclude node_modules && jscs *.js",
+    "lint": "eslint --fix .",
```

Comment:
I hadn't thought of enabling the auto-fix option (I don't do that in my own personal projects), but I guess I'm fine with it.